build/azure-pipelines/product-npm-package-validate.yml

```diff
@@ -0,0 +1,96 @@
+trigger: none
+
+pr:
+  branches:
+    include: ["main"]
+  paths:
+    include: ["package.json", "package-lock.json"]
+
+variables:
+  - name: Codeql.SkipTaskAutoInjection
+    value: true
+  - name: skipComponentGovernanceDetection
+    value: true
+  - name: NPM_REGISTRY
+    value: "https://pkgs.dev.azure.com/monacotools/Monaco/_packaging/vscode/npm/registry/"
```

Comment:
The value is configured in the pipeline, any reason to duplicate it here ?

Reply:
This pipeline doesn't reuse/inherit product-build pipeline thus we explicitly list it here.